### PR TITLE
Simplified build instructions

### PR DESCRIPTION
**Pull request by posix4e**
This simplifies the build instruction and assumes a modern enough system to support lsb_release

### CODE REVIEW

File: docs/linuxInstall.md [hunk 1]
```diff
@@ -1,44 +1,16 @@
...
-```
-wget -O brave.deb https://laptop-updates.brave.com/latest/dev/debian64
-sudo apt-get install -y gdebi && sudo gdebi brave.deb
-```
-
-## Ubuntu AMD64:
+## Debian or Ubuntu AMD64:
 To install brave using apt:
 ``` 
 curl https://s3-us-west-2.amazonaws.com/brave-apt/keys.asc | sudo apt-key add -
```
Comment (diracdeltas) [inline docs/linuxInstall.md]: not part of this PR but i had to do `curl https://s3-us-west-2.amazonaws.com/brave-apt/keys.asc 2> /dev/null | sudo apt-key add -` to prevent curl from spamming the output

**Comment by diracdeltas** [comment]
after following these instructions on debian jessie, i ran into an error on `apt-get update`:

```
W: The repository 'https://s3-us-west-2.amazonaws.com/brave-apt `lsb_release Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch https://s3-us-west-2.amazonaws.com/brave-apt/dists/`lsb_release/-cs`/binary-i386/Packages  403  Forbidden
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by posix4e** [comment]
@diracdeltas Sorry I missed your comment. We dont' support 32bit

File: docs/linuxInstall.md (hunk 1 above)
Comment (leighmcculloch) [inline docs/linuxInstall.md]: Shouldn't matter of the output is spammed because only stdout not stderr should be piped.

**Comment by posix4e** [comment]
github autoclosed this when i pushed :deb-docs

File: docs/linuxInstall.md (hunk 1 above)
Comment (diracdeltas) [inline docs/linuxInstall.md]: @leighmcculloch noted, but it looks confusing because it interferes with the sudo password prompt
